packages/utils/upgrade/resources/transforms/4.16.0/console.error-to-console.debug.code.ts

```diff
@@ -0,0 +1,37 @@
+import type { Transform } from 'jscodeshift';
+
+/**
```

Comment:
Could we just add an "examples" folder and leave it here forever? Or an ".examples." suffix to be ignored? Would be nice for people to have some things to look at where we could add extra comments explaining everything. If it's thorough enough (and this one is) that could practically count as the documentation.